src/theme/themes/plh_kids_kw/_overrides.scss

```diff
@@ -1153,4 +1210,93 @@ body[data-theme="plh_kids_kw"] {
       }
     }
   }
+
+  // Plh Progress Path
+  plh-plh-progress-path {
```

Comment:
nit(blocking): All this styling should live in `packages/components/plh/plh-kids-kw/components/plh-progress-path/plh-progress-path.component.scss`. As the component is already scoped to the deployment, I think that's the best place for the styling

Reply:
I think that's true for theme-based styling of a component that exists already, but in this case the component has no general styling currently, so there's no sense it which it is flexible across themes.

Also FYI, I've just pushed a minor update to this PR's branch and am going to target this branch for a build as per Laurie's email request, so pushing further commits to this branch today if possible